desktop/linux: Emphasise requirement for Gnome extension to support t… (#14765)
docker/docker.github.io: desktop/linux/install.md

- systemd init system.

- Gnome or KDE Desktop environment.

>   - On many distros the Gnome environment does not support tray icons. In such cases you will need to install a Gnome extension to add support for tray icons (e.g. [AppIndicator](https://extensions.gnome.org/extension/615/appindicator-support/)).

Reviewer: ```suggestion
  -For many Linux distros, the Gnome environment does not support tray icons. To add support for tray icons, you need to install a Gnome extension. For example, [AppIndicator](https://extensions.gnome.org/extension/615/appindicator-support/)).
```
